src/client/client4.js

```diff
@@ -225,6 +225,10 @@ export default class Client4 {
         return `${this.getPluginsRoute()}/${pluginId}`;
     }
 
+    getTimezonesRoute() {
+        return `${this.getBaseRoute()}/timezone`;
```

Comment:
Slight change, could we make the route `/api/v4/system/timezone` instead of `/api/v4/timezone`? Just since it's part of the system configuration and it would match better with our other endpoints